File: app/src/main/java/ir/example/model/VoiceModel.java
package ir.example.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;
import io.realm.annotations.Required;

@RealmClass(name = "Voice")
public class VoiceModel extends RealmObject {
    @PrimaryKey
    public long id;
    @Required
    public long termID;
    public long lessonID = -1;
    public long session = -1;
    @Required
    public String name;
    @Required
    public String path;


}

File: app/src/main/java/ir/example/model/VoiceModel.kt
package ir.example.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass
import io.realm.annotations.Required

@RealmClass(name = "Voice")
open class VoiceModel : RealmObject() {
    @PrimaryKey
    var id: Long = 0
    var termID: Long = 0
    var lessonID: Long = -1
    var session: Long = -1
    var name: String = ""
    var path: String = ""
}